tests/Fixer/ReturnNotation/ReturnAssignmentFixerTest.php

```diff
@@ -1165,6 +1165,20 @@ function add($a, $b): mixed
                 }
                 ',
             ],
+            'try finally2' => [
```

Comment:
```suggestion
            'try with multiple catch blocks' => [
```

There's no `finally` here 😉. And cases should be described in a way that intention is clear.